Example/Segmentify-Demo/AppDelegate.swift

```diff
@@ -41,7 +41,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         }
         
         application.registerForRemoteNotifications()
-         SegmentifyManager.config(appkey: "93a9a7c4-dc2e-49f7-832e-7c7fc3dabbf9", dataCenterUrl: "https://dce-test.segmentify.com", subDomain: "ihalilaltun.me")
+         SegmentifyManager.config(appkey: "da3d5f6f-63bf-48e5-ac6b-3b62585a824f", dataCenterUrl: "https://dce1.segmentify.com", subDomain: "renklidolap.com")
```

Comment:
@ebruersoy I am not sure, whether it is good idea to use production account for tests. You can use demo-test account for this matter.

Reply:
@candanceylan I had to use production account due to the connection problems of network. I could not call the gandalf test endpoints from ios, maybe because of the authorization. Also @mesutcank knows this situation. If you know any solution to handle this, we can solve it.

Reply:
Fair enough :)